base/src/test/java/com/mindera/skeletoid/generic/UIUtilsUnitTests.kt

```diff
@@ -0,0 +1,58 @@
+package com.mindera.skeletoid.generic
+
+import android.content.Context
+import android.content.res.Resources
+import com.mindera.skeletoid.generic.UIUtils.getStatusBarHeighPixels
+import org.junit.Assert
+import org.junit.Test
+import org.mockito.ArgumentMatchers
+import org.mockito.Mockito
+
+class UIUtilsUnitTests {
+    @Test
+    fun testsGetStatusBarHeightInvalid() {
+        val context =
+            Mockito.mock(Context::class.java)
+        val resources = Mockito.mock(
+            Resources::class.java
+        )
+        val invalidResourceId = -1
+        Mockito.`when`(context.resources).thenReturn(resources)
+        Mockito.`when`(
+            resources.getIdentifier(
+                ArgumentMatchers.any(String::class.java),
+                ArgumentMatchers.any(
+                    String::class.java
+                ),
+                ArgumentMatchers.any(String::class.java)
+            )
+        ).thenReturn(invalidResourceId)
+        Assert.assertEquals(0, getStatusBarHeighPixels(context))
```

Comment:
And then we can simplify this function.

```suggestion
        val invalidResourceId = -1
        
        Mockito.`when`(context.resources).thenReturn(resources)
        Mockito.`when`(
            resources.getIdentifier(
                ArgumentMatchers.any(String::class.java),
                ArgumentMatchers.any(String::class.java),
                ArgumentMatchers.any(String::class.java)
            )
        ).thenReturn(invalidResourceId)
        
        Assert.assertEquals(0, getStatusBarHeighPixels(context))
```